spring-integration-ip/src/main/java/org/springframework/integration/ip/udp/UnicastReceivingChannelAdapter.java

```diff
@@ -73,6 +73,11 @@ public UnicastReceivingChannelAdapter(int port, boolean lengthCheck) {
 	}
 
 	@Override
+	public boolean isLongLived() {
+		return true;
+	}
```

Comment:
This is now the only subclass (and likely will always be); but I didn't want to preclude that; we could pull down the `implements SAR` if you prefer.